dashboard/api_test.py

```diff
@@ -458,22 +457,21 @@ def test_not_paid_in_past_grade_frozen(self, froz_mock, upgr_mock, final_grades_
         )
 
         # case when the user has not an entry in the frozen grades
-        final_grades_mock.side_effect = FinalGrade.DoesNotExist
+        self.mmtrack.configure_mock(**{'extract_final_grade.side_effect': FinalGrade.DoesNotExist})
```

Comment:
This can be more clearly written as:
```python
self.mmtrack.extract_final_grade.side_effect = FinalGrade.DoesNotExist
```

You never need to use `configure_mock()` if you're only setting one thing in the mock. Just set it directly. (This also applies for all the `configure_mock()` calls below.)